source/features/sync-pr-commit-title.tsx

```diff
@@ -16,10 +17,6 @@ function getCommitTitleField(): HTMLInputElement | undefined {
 	return select<HTMLInputElement>(`${mergeFormSelector} #merge_title_field`);
 }
 
-function getPRNumber(): string {
```

Comment:
I think you should just export it from here.

Also this is not domReady safe

Reply:
> I think you should just export it from here.

Well we may need this in the future. It doesn't hurt.

> this is not domReady safe

What!? This is in the HTML!

![image](https://user-images.githubusercontent.com/44045911/111069071-4b074e00-8506-11eb-9289-c871eb44a038.png)

Reply:
use `getConversationNumber()`

Reply:
@yakov116 I'm confused, why do we have this function at all? `sync-pr-commit-title` runs on PRs too, so the number is in the URL

Reply:
That one needed the `#` so someone added it. I noticed it a while ago, but just left it.

Reply:
@yakov116 you stopped the blame too early. That line of code is ancient 😃 https://github.com/sindresorhus/refined-github/commit/ed2d06f0b0c441f16a7b444f3a8fc55e6f986924

Reply:
Ok I'll excuse you this time 😉